Dockerfile

```diff
@@ -45,7 +45,7 @@ COPY files/standalone.xml /opt/jboss/wildfly/standalone/configuration/standalone
 RUN mkdir -p /opt/utimaco/lib64 /opt/utimaco/p11
 ADD files/libcs_pkcs11* /opt/utimaco/p11/
 RUN \
-	ln -s libcs_pkcs11_R2.so /opt/utimaco/p11/libcs2_pkcs11.so && \
```

Comment:
You should be adding the R3 driver instead of replacing the existing one, so we can have compatibility with both engines.